src/staff-administrator/components/staff-add-edit-user/staff-user-check-answers/staff-user-check-answers.component.ts

```diff
@@ -1,20 +1,170 @@
 import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { StaffDataAccessService } from '../../../../staff-administrator/services/staff-data-access/staff-data-access.service';
+import { StaffFilterOption } from '../../../models/staff-filter-option.model';
+import { StaffJobTitles } from '../../../models/staff-job-titles';
 import { Router } from '@angular/router';
 import {
   FilterService
 } from '@hmcts/rpx-xui-common-lib';
+
 @Component({
   selector: 'exui-staff-user-check-answers',
   templateUrl: './staff-user-check-answers.component.html',
   styleUrls: ['./staff-user-check-answers.component.scss']
 })
+
 export class StaffUserCheckAnswersComponent implements OnInit {
-  private formId: string = 'staff-add-edit-user';
+  public formId: string = 'staff-add-edit-user';
+  public addUserData: {
+    name: string;
+    value: string[];
+  }[];
+  public staffFilterOptions: {
+    userTypes: StaffFilterOption[],
+    jobTitles: StaffFilterOption[],
+    skills: StaffFilterOption[],
+    services: StaffFilterOption[]
+  };
+  public firstName: string;
+  public lastName: string;
+  public email: string;
+  public region: string;
+  public services: string[];
+  public primaryLocations;
+  public additionalLocations;
+  public userType: string;
+  public roles: string[];
+  public skills: string[];
+  public servicePayload;
+  public rolesPayload;
+  public jobTitlesPayload = [];
+  public skillsPayload = [];
 
-  constructor(private filterService: FilterService,
-              private router: Router) { }
+  constructor(
+    private filterService: FilterService,
+    private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private staffDataAccessService: StaffDataAccessService
+  ) {
+    this.staffFilterOptions = {
+      userTypes: this.activatedRoute.snapshot.data.userTypes,
+      jobTitles: this.activatedRoute.snapshot.data.jobTitles,
+      skills: this.activatedRoute.snapshot.data.skills,
+      services: this.activatedRoute.snapshot.data.services
+    };
+  }
 
   public ngOnInit() {
+    this.filterService.getStream(this.formId).subscribe(data => {
+      this.addUserData = data.fields;
+      this.firstName = this.addUserData[0].value[0];
```

Comment:
Please have an `if` condition to check the value of `this.addUserData` to avoid run time error.